data-migration/migrate-mysql-tidb-above-tb.md

```diff
@@ -0,0 +1,273 @@
+    enable-gtid: true
+
+    from:
+      host: "${host}"           # e.g.: 172.16.10.81
+      user: "root"
+      password: "${password}"   # Supported but not recommended to use plaintext password. It is recommended to use `dmctl encrypt` to encrypt the plaintext password before using it.
+      port: 3306
+    ```
+
+2. Load the data source configuration to the DM cluster using `tiup dmctl` by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup dmctl --master-addr ${advertise-addr} operate-source create source1.yaml
+    ```
+
+    The parameters used in the command above are described as follows:
+
+    |Parameter              |Description    |
+    |-                      |-              |
+    |`--master-addr`        |The {advertise-addr} of any DM-master in the cluster where `dmctl` is to be connected, e.g.: 172.16.10.71:8261|
+    |`operate-source create`|Loads the data source to the DM cluster.|
+
+### Add a replication task
+
+1. Edit the `task.yaml` file. Configure the incremental replication mode and the starting point of each data source:
+
+    {{< copyable "shell-regular" >}}
+
+    ```yaml
+    name: task-test                      # Task name. Must be globally unique.
+    task-mode: incremental               # Task mode. The "incremental" mode only performs incremental data replication.
+
+    # Configures the target TiDB database.
+    target-database:                     # The target database instance.
+      host: "${host}"                    # e.g.: 127.0.0.1
+      port: 4000
+      user: "root"
+      password: "${password}"            # It is recommended to use `dmctl encrypt` to encrypt the plaintext password before using it.
+
+    # Use block and allow lists to specify the tables to be replicated.
+    block-allow-list:                    # The collection of filtering rules that matches the tables in the source database instance.
+      bw-rule-1:                         # The block-allow-list configuration item ID.
+        do-dbs: ["${db-name}"]           # Name of databases to be replicated.
+
+    # Configures the data source.
+    mysql-instances:
+      - source-id: "mysql-01"            # Data source ID，i.e., source-id in source1.yaml
+        block-allow-list: "bw-rule-1"    # References the block-allow-list configuration above.
+        # syncer-config-name: "global"    # References the syncers incremental data configuration below.
+        meta:                            # When task-mode is "incremental" and the downstream database does not have a checkpoint, DM uses the binlog position as the starting point. If the downstream database has a checkpoint, DM uses the checkpoint as the starting point.
+          # binlog-name: "mysql-bin.000004"  # The binlog position recorded in Step 1. When the upstream database has source-replica switching, GTID mode is required.
```

Comment:
`If the upstream database service is configured to switch master between different nodes automatically, GTID mode is required`